docs/apidiffs/current_vs_latest/opentelemetry-instrumentation-api-annotation-support.txt

```diff
@@ -0,0 +1,2 @@
+Comparing source compatibility of  against 
+No changes.
```

Comment:
does this mean that this has no non-internal public classes/methods?

Reply:
This means that it did not change since the last release. At least I read id this way.

Reply:
Was this marked stable in a previous release, then?

Reply:
No, why?

Reply:
The gradle code is supposed to generate the full API in the case where there is no previously published stable release. If it's not doing that, then there might be something amiss without how the porting over from the java repo happened.

Reply:
A, indeed, I changed the code to search for `-alpha` version, because we don't have stable version yet. I am not sure which version is more useful. What benefit do you get from the full API listing?

Reply:
Having a baseline of the API can be useful when looking at the first official stable release, as a documentation of the actual API that is now stable, rather than just seeing "no changes".

Reply:
I see. And under which folder such full API listing should live? I suspect not under "current vs latest"?

Reply:
I removed my support for unstable versions and also removed this diffs, as they don't make sense for now.

Reply:
> I see. And under which folder such full API listing should live? I suspect not under "current vs latest"?

I think it would belong there if the 'current' has been marked stable in the current snapshot. It's a description of what you're preparing to release as a new stable API.

Reply:
> > I see. And under which folder such full API listing should live? I suspect not under "current vs latest"?
> 
> I think it would belong there if the 'current' has been marked stable in the current snapshot. It's a description of what you're preparing to release as a new stable API.

Got it, makes sense. So we will add that API list when we are ready to declare it stable.